src/hooks/use-sync-sites.ts

```diff
@@ -1,92 +1,126 @@
-import { useState, useEffect } from 'react';
+import * as Sentry from '@sentry/electron/renderer';
+import { useState, useEffect, useRef } from 'react';
+import { useAuth } from './use-auth';
+import { useOffline } from './use-offline';
 
-export type SyncSupport = 'unsupported' | 'syncable' | 'needs-transfer' | 'already-connected';
+const BUSINESS_PLAN_ID = 1008;
+const COMMERCE_PLAN_ID = 1011;
```

Comment:
> Let's make the Studio plan-agnostic if possible so it would only operate based on available features, e.g. atomic in this case.

Let's use `site.plan.features.active.includes('atomic')`. And if we want to limit the Pro plan, then we can add a condition for that specific plan.

If instead of using the site features, we prefer to use the plan ids, then we would need to add also all the bundles (monthly, 2-year, 3-year). Those have different plan IDs.